jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST

```diff
@@ -5,7 +5,7 @@ source "${HOMEgfs}/ush/preamble.sh"
 ##############################################
 # make temp directory
 ##############################################
-export DATA=${DATA:-${DATAROOT}/ocnanal_${CDATE}} # TODO (G): Switch to {cyc} when the downstream code is ready
+export DATA=${DATA:-${DATAROOT}/ocnanal_${PDY}${cyc}} 
```

Comment:
```suggestion
export DATA=${DATA:-${DATAROOT}/${RUN}ocnanal_${cyc}} 
```

Reply:
Isn't just using `${cyc}` going to introduce the possibility of collisions with other dates? I know the DA is serial, but if `${KEEPDATA}` is on, the old directory will still be sitting there.

Reply:
> Isn't just using `${cyc}` going to introduce the possibility of collisions with other dates? I know the DA is serial, but if `${KEEPDATA}` is on, the old directory will still be sitting there.

Good point @WalterKolczynski-NOAA . We'll need to have a `${KEEPDATA}` on switch that renames `${RUN}ocnanal_${cyc}}` to something with `PDY` in its name. We'll tackle this when we decide to actually do something in post!

Reply:
Why not just include PDY to begin with? Unclear why @aerorahul asked for it to be removed (`${RUN}` can be added without removing `${PDY}`).

Reply:
@aerorahul @WalterKolczynski-NOAA @guillaumevernieres Here's a problem with this - in the current soca test, RUN gets assigned in config.base, which comes after the DATA assignment in the jjob. Is this contrary to the assumptions within g-w?

Reply:
`$RUN` should be defined in the pre-job script when needed (`job/rocoto/*` or `ecf/scripts/*`), so it should be added to the rocoto pre-script if it is not already there (don't worry about ecf yet), and the test system should replicate that (if it isn't just using the rocoto pre-script).